IdentityServer/appsettings.json

```diff
@@ -33,6 +31,10 @@
             "FhictClientSecret": "",
             "FhictRedirectUri": "",
             "FhictScopes": ""
+        },
+        "Frontend": {
+            "RedirectUrisFrontend": [],
```

Comment:
Shouldn't these be "" instead of [] ?

Reply:
No, this is an array. Multiple redirect uri's can be given.